keyvault/2016-10-01/swagger/keyvault.json

```diff
@@ -158,7 +161,8 @@
           "Keys"
         ],
         "operationId": "UpdateKey",
-        "description": "The update key operation changes specified attributes of a stored key and can be applied to any key type and key version stored in Azure Key Vault. The cryptographic material of a key itself cannot be changed. In order to perform this operation, the key must already exist in the Key Vault. Authorization: requires the keys/update permission.",
+        "summary": "The update key operation changes specified attributes of a stored key and can be applied to any key type and key version stored in Azure Key Vault.",
+        "description": "The cryptographic material of a key itself cannot be changed. In order to perform this operation, the key must already exist in the Key Vault.",
```

Comment:
Its an Key update operation that takes the specified attributes as arguments and sets those attrs. What more would you like to see?